src/CppParser/Parser.cpp

```diff
@@ -63,12 +64,23 @@
 using namespace CppSharp::CppParser;
 
 // We use this as a placeholder for pointer values that should be ignored.
-void* IgnorePtr = (void*) 0x1;
+void* IgnorePtr = reinterpret_cast<void*>(0x1);
+
+static bool initialized_llvm = false;
```

Comment:
Please rename this to follow our convention, no underscores to separate words in identifiers.